pom.xml

```diff
@@ -61,12 +61,6 @@
         <version>5.4.2</version>
         <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <version>5.4.2</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
There a particular reason you're deleting these?

Reply:
Yes! There were in my pom file twice for some reason.